app/Config/Schema/sessions.php

```diff
@@ -44,14 +46,14 @@ public function before($event = array()) {
  * @param array $event Schema object properties
  * @return void
  */
-	public function after($event = array()) {
-	}
-
-	public $cake_sessions = array(
-		'id' => array('type' => 'string', 'null' => false, 'key' => 'primary'),
-		'data' => array('type' => 'text', 'null' => true, 'default' => null),
-		'expires' => array('type' => 'integer', 'null' => true, 'default' => null),
-		'indexes' => array('PRIMARY' => array('column' => 'id', 'unique' => 1))
-	);
+    public function after($event = array())
+    {
+    }
 
+    public $cake_sessions = array(
+        'id' => array('type' => 'string', 'null' => false, 'key' => 'primary'),
+        'data' => array('type' => 'text', 'null' => true, 'default' => null),
+        'expires' => array('type' => 'integer', 'null' => true, 'default' => null),
+        'indexes' => array('PRIMARY' => array('column' => 'id', 'unique' => 1))
+    );
```

Comment:
__[staging][code_sniffer]__
* Closing parenthesis not aligned correctly; expected 28 space(s) but found 4